modules/retail/pages/retail-best-practices.adoc

```diff
@@ -1,75 +1,64 @@
 [[retail-best-practices]]
-= Best Practices
+= Best practices
 
-== Deploying New Images
+== Deploying new images
 
 This file describes how to deploy a new image on selected terminals first, test it, and then deploy it to all terminals.
 
-The hardware type (HWTYPE) group is used for booting new terminals, while already deployed terminals can
-be moved to any group that provides the {saltboot} formula. 
-This flexibility allows for testing of new images
-or configurations. 
-To ensure a smooth deployment, follow these steps:
+The hardware type group [literal]``HWTYPE`` is used for booting new terminals, while already deployed terminals can be moved to any group that provides the {saltboot} formula. 
```

Comment:
This is sending different message then originally thought.
The intent was to provide some recognizable string for hardware type group.

In the new version it seems like we are talking about hardware type group named `HWTYPE` which is not the case. `HWTYPE` is just a prefix for the hardware type group name.